docs/reference/overview_of_strategies.rst

```diff
@@ -1123,3 +1123,13 @@ LookerUp
 
 LookerUp uses a lookup table to decide what to play based on the last few rounds plus
 the first few plays made by the opponent at the start of the match.
+
+ThueMorse
+^^^^^^^^^
+
+This strategy uses the [Thue-Morse sequence](http://mathworld.wolfram.com/Thue-MorseSequence.html) in order to determine its actions. It will Cooperate on a 1 and Defect on a 0.
```

Comment:
That's right, but it's not independent of the full history of play. In other words it doesn't matter what the last `n` opponent plays were but it does depend on the length.